mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -594,6 +598,8 @@ public void getAccount(GetAccountRequest request, StreamObserver<GetAccountRespo
                                 .addEntitlements(createEntitlement(DATAHUB_AZURE_AUTOSCALING))
                                 .addEntitlements(createEntitlement(DATAHUB_AWS_AUTOSCALING))
                                 .addEntitlements(createEntitlement(LOCAL_DEV))
+                                .addEntitlements(createEntitlement(DATAHUB_FLOW_SCALING))
+                                .addEntitlements(createEntitlement(DATAHUB_STREAMING_SCALING))
```

Comment:
Shouldn't the `DATAHUB_DEFAULT_SCALING` should also be added here?

Reply:
I dont want to enable that one yet

Reply:
ok